steps/container_steps.py

```diff
@@ -315,8 +321,10 @@ def run_command_expect_message(context, cmd, output_phrase, timeout=80):
                     return True
             except ExecException as e:
                 last_output = e.output
+                exception_cause = e
                 time.sleep(1)
-    raise Exception("Phrase '%s' was not found in the output of running the '%s' command" % (output_phrase, cmd), last_output)
+    raise Exception("Phrase '%s' was not found in the output of running the '%s' command" % (output_phrase, cmd),
```

Comment:
I don't think I quite understand the intent here (I get that you're using | true. What will happen to tests using this that are expecting to get the exception?

Reply:
this code change will not change the function's behavior, instead, just append more information on the exception printed to the user instead just tell that the expected text does not match the result.

>> What will happen to tests using this that are expecting to get the exception?
 I could be wrong, but, if it is possible, should behave the same way, but, if you want to check for e ExecException then you'll need to change the except logic to catch that.

Reply:
Well, the only reason I ask is if the behaviour changes and we have tests using it, they'll all need to be fixed, so it could be a better idea to add a new test method instead. If it won't break anything that exists already, then no need though.

Reply:
we have some tests that use it and it seems to be working ok. I'll give it another try and run the tests that use it individually.
On the other hand, do you mind pointing me to some tests that you are afraid to be affected so I can test it as well here just to be sure it will not break anything?

Reply:
There are some tests for in the jboss-eap-7-openshift-image and jboss-eap-modules repos, after a brief look they appear to all expect to return something sensible though, so not sure if this change would have an impact if say the command expected was no longer there or something.

Reply:
this will not change the output or method return, is there a way for you to give it a try with my v1 branch?

Reply:
@spolti Can this be merged?